bin/lib/fastly-extended.js

```diff
@@ -39,7 +39,14 @@ module.exports = function (apiKey, serviceId) {
             }
             var latestVersion = versions.reduce(function (lateVersion, version) {
```

Comment:
I'm surprised our linter didn't warn/error on this. Don't we prefer (lateVersion, version) => { ... } ? Also, the var statements here ought to be const's. But that is pre-existing, so a new issue and not this PR.

Reply:
Also to avoid confusion, I think the two objects being compared in the reduce() function ought not be named the same as the object being returned via callback at line 52 and defined/getting return from the reduce() at line 40.

Reply:
(sorry, just noticed they're not the same name, just close named)